2018-02-26T05:04:20.055Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'supplier3' in collection with ID 'Participant:org.capita.Supplier' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T05:04:20.056Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'supplier3' in collection with ID 'Participant:org.capita.Supplier' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'supplier3' in collection with ID 'Participant:org.capita.Supplier' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-02-26T05:04:46.886Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'supplier2' in collection with ID 'Participant:org.capita.Supplier' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T05:04:46.886Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'supplier2' in collection with ID 'Participant:org.capita.Supplier' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'supplier2' in collection with ID 'Participant:org.capita.Supplier' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-02-26T05:08:13.451Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T05:08:13.452Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)","stack":"Error: Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)\n    at channel.queryByChaincode.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-02-26T06:20:00.300Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T06:20:00.333Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)","stack":"Error: Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)\n    at channel.queryByChaincode.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-02-26T08:19:33.615Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T08:19:33.615Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)","stack":"Error: Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)\n    at channel.queryByChaincode.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-02-26T15:31:17.274Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T15:31:17.275Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)","stack":"Error: Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)\n    at channel.queryByChaincode.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-02-26T15:34:39.405Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T15:34:39.405Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)","stack":"Error: Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)\n    at channel.queryByChaincode.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-02-26T15:39:55.098Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'uid224' in collection with ID 'Participant:org.capita.Consumer' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T15:39:55.099Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'uid224' in collection with ID 'Participant:org.capita.Consumer' does not exist)","stack":"Error: Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'uid224' in collection with ID 'Participant:org.capita.Consumer' does not exist)\n    at channel.queryByChaincode.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-02-26T15:41:34.468Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)
    at new createStatusError (/usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:64:15)
    at /usr/lib/node_modules/composer-rest-server/node_modules/grpc/src/client.js:583:15 {}$
2018-02-26T15:41:34.470Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)","stack":"Error: Error trying to query business network. Error: 2 UNKNOWN: chaincode error (status: 500, message: Error: Object with ID 'undefined' in collection with ID 'Participant:org.capita.Consumer' does not exist)\n    at channel.queryByChaincode.then.catch (/usr/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
